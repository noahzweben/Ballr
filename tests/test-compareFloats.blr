gameboard g1{
	clr = (251,142,74);
	size = (680,420);
	init -> {

		float x = 1.2;
		float y = 3.5;

		if (x > y) {
			print(0);
		}

		if (x < y) {
			print(1);
		}

		if (x >= x) {
			print(1);
		}

		if (y <= x){
			print(0);
		}

		if (y != x) {
			print(1);
		}

		if (y == y){
			print(1);
		}
  		
  	}
}