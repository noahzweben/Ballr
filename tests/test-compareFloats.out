1
1
1
1
SDL could not initialize! SDL Error: No available video device
Window initialization failed.